interface/billing/search_payments.php

```diff
@@ -468,13 +468,13 @@ function SearchPayingEntityAction()
           </tr>
           <tr>
             <td align="right"></td>
-			<td align="left" class="text"><?php echo htmlspecialchars( xl('Payment From'), ENT_QUOTES).':' ?></td>
+			<td align="left" type="text" class="text"><?php echo htmlspecialchars( xl('Payment From'), ENT_QUOTES).':' ?></td>
```

Comment:
Please note that when this is rendered in the browser it is an input box hence type="text", helps style element